docs/guide/java_object_graph_guide.md

```diff
@@ -94,7 +94,7 @@ System.out.println(fury.deserialize(bytes));
 ```
 
 ### Security & Class Registration
-`FuryBuilder#requireClassRegistration`/`FuryBuilder#withSecureMode` can be used to disable class registration, this will allow to deserialize objects unknown types, more flexible but **less secure**.
+`FuryBuilder#requireClassRegistration` can be used to disable class registration, this will allow to deserialize objects unknown types, more flexible but **less secure**.
```

Comment:
Perhaps we can elaborate a bit why it's "less secure" and thus users can check if their env is "indeed secure".

Reply:
Good suggestion, added detailed insecurity reason